src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -306,6 +308,18 @@ private IEnumerable<PackageSource> LoadNuGetSources(PackageSourceLocation packag
             PackageSourceProvider packageSourceProvider = new PackageSourceProvider(settings);
             defaultSources = packageSourceProvider.LoadPackageSources().Where(source => source.IsEnabled);
 
+            PackageSourceMapping packageSourceMapping = PackageSourceMapping.GetPackageSourceMapping(settings);
```

Comment:
The NuGetPackageDownloader is also used to download workload NuGet packages.  We don't yet want to force package source mapping on those downloads.  So we should add a constructor argument or a property to this class so that package source mapping can be enabled for tool installs but not workload package downloads.

@nkolev92 @zivkan FYI on this, we don't want to enable PSM for workload package downloads this late in the .NET 8 release, as we think it would be disruptive.  Do you think we should try to enable it for workload packages in a later release?

Reply:
The purpose of package source mapping is to give customers control over supply chain security concerns. So, I'd say yes.

Reply:
@dsplaisted Sounds good. Are there properties that can be used to differentiate a tool from workloads? Any examples would be appreciated:)